resources/electron-preload.js

```diff
@@ -0,0 +1,25 @@
+(function(){
+  'use strict';
+  const {ipcRenderer} = require('electron');
```

Comment:
This code is only ran in Electron. I don't think anyone builds desktop apps with old Electron versions running on Node 4.